mlflow/pyfunc/utils/environment.py

```diff
@@ -0,0 +1,20 @@
+import os
+from contextlib import contextmanager
+
+from mlflow.environment_variables import _MLFLOW_IS_SERVING_ENVIRONMENT
+
+
+@contextmanager
+def serving_environment():
```

Comment:
q: Is this method meant to be public and used by users?

Reply:
Let me update it to _serving_environment :)